uks/apps/repository/views.py

```diff
@@ -103,3 +109,71 @@ def add_repository(request):
         form = RepositoryForm()
 
     return render(request, 'user/dashboard.html', {'form': form})
+
+
+def RepositorySettings(request, id):
+
+    repository = Repository.objects.get(id=id)
+    global repo
+    repo = repository.id
+    users = User.objects.filter().exclude(id=repository.owner.id).exclude(username='admin')
+    collabs = repository.collaborators.all()
+    context = {'repository': repository, 'users': users, 'collabs': collabs}
+    return render(request, 'repository/repoSettings.html', context)
+
+
+def addCollaborators(request):
+    if request.method == 'POST':
+        form = CollaboratorsForm(request.POST)
+        if form.is_valid():
+            users = form.data.getlist('collaborators')
+            global repo
+            repository = Repository.objects.get(id=repo)
+            for u in users:
+                user = User.objects.get(username=u)
+                Repository.objects.get(id=repo).collaborators.add(user)
+
+        else:
+            print('Forma nije validna')
+
+    else:
+        form = CollaboratorsForm()
+
+    repository = Repository.objects.get(id=repo)
+    users = User.objects.filter().exclude(id=repository.owner.id).exclude(username='admin')
+    collabs = repository.collaborators.all()
+    context = {'repository': repository, 'users': users, 'collabs': collabs}
+    return render(request, 'repository/repoSettings.html', context)
+
+
+# Bug brise iz baze celog usera
```

Comment:
If this bug is still active probably the reason is cascade delete. Check on_delete property for attribute user